cmd/minikube/cmd/tunnel.go

```diff
@@ -54,6 +57,7 @@ var tunnelCmd = &cobra.Command{
 		RootCmd.PersistentPreRun(cmd, args)
 	},
 	Run: func(cmd *cobra.Command, args []string) {
+		defer cleanupLock()
```

Comment:
Can you plz also put output of minikube when cluster is not running ? To ensure this wont crash

Reply:
Updated PR description with the output. Let me know if that is not what you were expecting!